lib/elm-test.js

```diff
@@ -133,6 +133,17 @@ function main() {
       'Use a custom path to an Elm executable (default: elm)',
       undefined
     )
+    .addOption(
+      new Option(
+        "--output <output>",
+        "Allowed for compatibility with `elm make`. The only supported value is /dev/null, which elm-test ignores since it does not produce any output files."
```

Comment:
Just curious: Is there any way of making this text show up?

Do we want to document – maybe as a comment – that it’s for Emacs users who use `elm make` for source files, and `elm-test make` for test files, and Emacs-something always adds on `--output /dev/null`?

Reply:
> Just curious: Is there any way of making this text show up?

I don't think so 🤔 

> Do we want to document – maybe as a comment – that it’s for Emacs users who use elm make for source files, and elm-test make for test files, and Emacs-something always adds on --output /dev/null?

Will do!